### ISSUE TITLE
Switch Audio and Recording device with same hotkey

### ISSUE THREAD

**Issue by Racle** [Belphemur/SoundSwitch#409]
Ex. I have this situtation. 
I use VR headset (Oculus Rift S) and it has speaker and microphone. After I'm done playing with that, I usually switch manually both speaker and microphone to another devices. And at boot Rift S usually "steals" speaker and microphone settings, so I need manually switch to my another devices.

In SoundSwitch, I can do that with two hotkeys. Toggle between those devices/profiles would be nice to have.

**Comment by Belphemur** [comment]
Hello, 

This is already possible with the profile feature. 

Go into settings and setup a profile with the playback and recording device + the wanted hotkey. 

There you go, you can now swtich directly to that profile using the set hotkey. 

Sent from Nine
________________________________

**Comment by Racle** [comment]
Hey,

Not possible to toggle between those using one keybind, which was the enhancement point.

Now I need to use two keybindings with two profiles instead of one "toggle between profiles" keybind (AKA this enhancement).

Toggle same way you can toggle between sound devices or recording devices currently. And if I try to use same keybind on those, it just toggles between sound devices OR recording devices. Not both.

**Comment by Belphemur** [comment]
Sorry to come back so late.

I see the use case, switching between 2 profiles or more likely having a list of devices per type in the profile itself.

This way, you could create "group" in the same profile and use the same hotkey to cycle through them.

**Comment by tonyrobots** [comment]
Yes, I would like this as well. Use case (like many others, I think) is switching between wireless headset (with mic) and normal speakers/onboard mic. Can set each in a profile now and activate with two different hotkeys, but would prefer to be able to toggle between those configurations with just one hotkey.

Another simple approach without involving profiles would for both output and recording devices to be switchable with the same hotkey.

**Comment by Racle** [comment]
> Another simple approach without involving profiles would for both output and recording devices to be switchable with the same hotkey.

This is possible currently :)

I use ctrl+shift+F1-3 to switch between my 3 profiles (all which have different speaker+microphone combo). 
It's working perfectly for me 👌

**Comment by vertigo220** [comment]
What's being asked is for the ability to use a _single_ hotkey to toggle/cycle vs a different hotkey for each config. There are multiple situations where having one hotkey is more useful, and I just found this app after getting tired of constantly having to manually switch between onboard audio and my headset and want to use a special key on my keyboard to toggle it, but that means setting it to use _one_ hotkey.

Furthermore, I found what appears to be a bug when setting both the playback device and recording device hotkey to the same thing (which is how I figured I could get it to work the way I wanted). It would only switch the playback device, despite both being set to that hotkey, and even if I disable the hotkey on the playback tab, it still toggles the playback device and not the recording device, even though that's the only one enabled.

**Comment by Belphemur** [comment]
I'm adding this feature in the next version.

By setting multiple profile with the same hotkey, you'll be able to switch between them.

**Comment by Belphemur** [comment]
Full explanation of the feature (to be released):
https://github.com/Belphemur/SoundSwitch/discussions/738

**Comment by Belphemur** [comment]
Also the latest Beta has the feature now:
https://nc.aaflalo.me/s/8DKytYCTAbrn6yz

**Comment by supleed2** [comment]
Can we also have the option to disable the quick menu? Similar to the check box for the quick menu when using a hotkey to change the recording/playback device individually.
![option](https://user-images.githubusercontent.com/21363892/132136426-0eda5eba-603d-4304-800e-977fb1ad8855.png)

**Comment by vertigo220** [comment]
Thanks! There does still seem to be a problem with it. I set one profile for onboard speakers and mic and another for headset, and when I switch, the pop-up (banner) says "Profile...triggered" and shows the appropriate devices, but the second pop-up next to the cursor (which seems redundant and I think may be the one to which @supleed2 is referring) shows the _opposite_ output device selected, and hovering over or clicking on the tray icon, the pop-up there shows the correct microphone but the wrong output device. I tried manually setting them the same in case it was simply toggling them instead of actually specifically selecting them, so they would be synced, but as soon as I activate the other profile, they're off again. It's always the output device that's wrong.

Adding to the confusion, after playing around with it trying to figure it out, now the second pop-up (next to the cursor) shows the same device as the banner pop-up, i.e. the correct device, but the tray icon tooltip still shows the opposite one, and that's the one that's actually active.
